Pipelines/Templates/set-website-variable.yml

```diff
@@ -0,0 +1,38 @@
+#We are going to set websiteId as a pipeline variable only if there is a portal in the Dataverse
+
+parameters:
+- name: serviceConnectionUrl
+  type: string
+  #In case of exporting portal, we expect that the solution unique name will be the same as the portal name
+- name: solutionName
+  type: string
+
+steps:
+- powershell: | 
+    $websiteId = ${env:WEBSITEID}
```

Comment:
@mawasile Where is the WEBSITEID variable declared? I might be missing something. FYI @mikefactorial

Reply:
@RajeevPentyala this variable is there to leave the user a possibility to inject his websiteid using this variable, in case if would like to, then we will not look for the websiteid ourselves.  In normal circumstances $websiteId will start and $null as this is not set in the pipelines.